roles/glusterfs_server/tasks/130_gluster_add_bricks.yaml

```diff
@@ -0,0 +1,23 @@
+- set_fact:
+    cluster_hosts: "{{ play_hosts | join(',') }}"
+
+- debug: var=cluster_hosts
+
+- name: create gluster volume
+  gluster_volume:
+    state: present
+    name: "{{ glusterfs_volume_name }}"
+    bricks: "{{ glusterfs_storage_dir }}"
+    cluster: "{{ cluster_hosts }}"
+    force: yes
+    rebalance: yes
+    transport: tcp
+    replicas: "{{ num_hosts  }}"
+    options:
+      storage.owner-uid: "105"
+      storage.owner-gid: "108"
+  run_once: True
+  register: result
+  until: result|succeeded
+  retries: 5
+  delay: 10
```

Comment:
After investigation on this task: it is possible to increase/decrease number of bricks, but doing in automated way has high risks of bringing gluster cluster to failed state